test/e2e/endpointslices/topology.go

```diff
@@ -86,7 +85,6 @@ var _ = framework.IngressNginxDescribeSerial("[TopologyHints] topology aware rou
 		}
 
 		if gotHints {
-			//we have 2 replics, if there is just one backend it means that we are routing according slices hints to same zone as controller is
```

Comment:
Why the comment was removed? We need it to understand sometimes why some tests are being executed :)

Reply:
Revert it